modules/kubernetes/linkerd/main.tf

```diff
@@ -193,10 +193,10 @@ resource "kubernetes_secret" "webhook_issuer_tls" {
 #tf-latest-version:ignore
 resource "helm_release" "linkerd_cni" {
   repository  = "https://helm.linkerd.io/stable"
-  chart       = "linkerd2-cni"
+  chart       = "linkerd-control-plane"
   name        = "linkerd-cni"
   namespace   = kubernetes_namespace.cni.metadata[0].name
-  version     = "2.11.3"
+  version     = "30.3.0"
```

Comment:
Note the versions follows some semver of the helm chart and not the app version any more. 30.3.0 is 2.12.0